### PR TITLE
Fix version

### PR DESCRIPTION
**Pull request by botelhoa**
Version file currently being pulled in from `version.txt` which does not exist. I replaced it with the latest version number in pypi

### CODE REVIEW

**Review by c-w — COMMENTED** [review]
Oh how strange. Good catch! Could you re-create the `version.txt` with content `0.1.0`? Just so we don't need to also update `MANIFEST.in` etc. Thanks in advance!

**Comment by nzola** [comment]
Thanks Austin for the catch and thanks Clemens for responding to Austin!

**Comment by botelhoa** [comment]
Done!

File: version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+"0.1.0"
```
Comment (c-w) [inline version.txt]:
```suggestion
0.1.0
```

**Review by c-w — CHANGES_REQUESTED** [review]

File: version.txt (hunk 1 above)
Comment (botelhoa) [reply version.txt]: String quotes removed

**Review by c-w — APPROVED** [review]

**Pull request merged**